client/src/api/configurations/configurations.ts

```diff
@@ -674,4 +676,83 @@ export const useDeleteCustomCodeFromConfiguration = <TError = AxiosError<HTTPVal
 
       return useMutation(mutationOptions , queryClient);
     }
+    /**
+ * Runs an in-line test using the provided configuration ID and test files.
+
+Args:
+    id (UUID): ID of Configuration to use for the test
+    uploaded_file (UploadFile | None): user uploaded eICR/RR pair
+    create_output_zip (Callable[..., tuple[str, io.BytesIO]]): service to create an in-memory zip file
+    upload_refined_files_to_s3 (Callable[[UUID, io.BytesIO, str, Logger], str]): service to upload a zip to S3
+    user (DbUser): Logged in user
+    db (AsyncDatabaseConnection): Database connection
+    sample_zip_path (Path): Path to example .zip eICR/RR pair
+    logger (Logger): Standard logger
+Returns:
+    ConfigurationTestResponse: response given to the client
+ * @summary Run Configuration Test
+ */
+export const runInlineConfigurationTest = (
```

Comment:
is this function codegened at all, or is it just taking the generated types from Orval to facilitate the different testing pieces? Trying to grok how the different parts fit together

Reply:
yep saw that page so wanted to make sure that's what was happening here. thanks so much!